crates/l2/tests/tests.rs

```diff
@@ -1592,10 +1622,17 @@ async fn test_total_eth_l2(
 
     println!("Bridge locked ETH: {bridge_locked_eth}");
 
-    assert!(
-        total_eth_on_l2 <= bridge_locked_eth,
-        "Total ETH on L2 ({total_eth_on_l2}) is greater than bridge locked ETH ({bridge_locked_eth})"
-    );
+    if std::env::var("INTEGRATION_TEST_SKIP_FEE_VAULT_CHECK").is_err() {
+        assert!(
+            total_eth_on_l2 == bridge_locked_eth,
+            "Total ETH on L2 ({total_eth_on_l2}) differs from bridge locked ETH ({bridge_locked_eth})"
+        );
+    } else {
+        assert!(
+            total_eth_on_l2 < bridge_locked_eth,
```

Comment:
Shouldn't here be `<=`?

```suggestion
            total_eth_on_l2 <= bridge_locked_eth,
```